addons/hr_attendance/report/hr_attendance_report.py

```diff
@@ -0,0 +1,51 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+from odoo import fields, models, tools
+
+
+class HRAttendanceReport(models.Model):
+    _name = "hr.attendance.report"
+    _description = "Attendance Statistics"
+    _auto = False
+
+    department_id = fields.Many2one('hr.department', string="Department", readonly=True)
+    employee_id = fields.Many2one('hr.employee', string="Employee", readonly=True)
+    check_in = fields.Date("Check In", readonly=True)
+    worked_hours = fields.Float("Hours Worked", readonly=True)
+    overtime_hours = fields.Float("Extra Hours", readonly=True)
+
+    def init(self):
+        tools.drop_view_if_exists(self.env.cr, self._table)
+
+        self.env.cr.execute("""
+            CREATE OR REPLACE VIEW %s AS (
```

Comment:
isn't combination of both a bit overkill?

Reply:
I agree it's redundant, though it is present in a lot of report in the codebase, I will remove it.